### DISCUSSION TITLE
Monitor with negative readings

### DISCUSSION THREAD

**Discussion by walinchus** [USEPA/RAQSAPI#20] [Q&A] [answered]
In Palm Beach County – Lamstein Lane Site Monitor #12-099-0022 has several negative readings for PM 2.5. 

<img width="657" alt="image" src="https://github.com/USEPA/RAQSAPI/assets/25959269/19369017-50f4-42e4-aad9-173fee8f7875">

I'm not seeing this with any other monitor in South Florida: 
<img width="1169" alt="image" src="https://github.com/USEPA/RAQSAPI/assets/25959269/6ff47425-6a48-4ceb-b761-b9a87959a499">

Are the readings supposed to be positive? Was an error code supposed to be attached to the readings?

Thank you for your help in this matter. And fantastic package.

**Comment by mccroweyclinton-EPA** [comment, marked as answer]
Thank you for your interest in RAQSAPI. 

Quality assurance with regard to environmental data is a very complicated matter. The US EPA does allow negative values for PM measurements in AQS. Negative values for PM measurements could be caused by random instrumentation noise on an air samples that does not have any or a very small amount of measurable PM.

Please refer to this [powerpoint presentation that briefly discusses this very concept, on slide 7](https://www.epa.gov/sites/default/files/2016-10/documents/pm2.5_continuous_monitoring.pdf)

For future reference please post questions on the discussions tab, and only post issues with the code itself on the issue tab. I have converted this issue into a discussion.

**Reply by walinchus** [reply]
Oh sure will do. So the negative number is essentially zero?

**Reply by mccroweyclinton-EPA** [reply]
"essentially" =)

**Reply by walinchus** [reply]
Okay thanks
